### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
#Summary
add navbar

#Screenshot
captura de pantalla 2018-09-11 a la s 14 33 07

##Trello Card
https://trello.com/c/a1JdMQDB/13-navbar

### CODE REVIEW

File: scss/color.scss [hunk 1]
```diff
@@ -1,2 +1,3 @@
 $color-black: #000;
 $color-green: #BED23C;
+$color-white: #fff;
```
Comment (mmamaniwolox) [inline scss/color.scss]: Use uppercase here #FFF

File: scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 1;
+}
+
+.image-navbar {
+  width: 120px;
```
Comment (mmamaniwolox) [inline scss/navbar.scss:18]: Use height or width, that will be enought

File: scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+@import 'color';
+
+.navbar {
+  position: fixed;
+  background: $color-white;
+
```
Comment (mmamaniwolox) [inline scss/navbar.scss]: Delete this empty line

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -8,7 +8,12 @@
...
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body class ="body">
+<body>
+  <nav class= "navbar">
+    <img class="image-navbar" src="images/wolox_logo.svg" alt="Wolox logo">
+    <h1 class="title-navbar">Developer Tools</h1>
+  </nav>
+  <div class="body">
   <h1 class="title">Front-End Tools</h1>
```
Comment (mmamaniwolox) [inline index.html]: You miss a tabbing in all div body

File: index.html [hunk 5]
```diff
@@ -8,7 +8,12 @@
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body class ="body">
+<body>
+  <nav class= "navbar">
+    <img class="image-navbar" src="images/wolox_logo.svg" alt="Wolox logo">
+    <h1 class="title-navbar">Developer Tools</h1>
+  </nav>
+  <div class="body">
```
Comment (mmamaniwolox) [inline index.html]: Use a different class name, page-contetnt for example. Body is an html element, this may be confusing

File: scss/navbar.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 1;
+}
+
+.image-navbar {
+  height: 80px;
+  width: 120px;
```
Comment (mmamaniwolox) [inline scss/navbar.scss]: Use only height or width

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

File: index.html [hunk 7]
```diff
@@ -8,54 +8,59 @@
...
-    <div class="techno-content">
-      <img class="img-techno" src="images/react_icon.svg" alt="React icon">
-      <p class="tech-text">ReactJS</p>
-      <p class="text-release">Release:2013</p>
-      <p class="text-complete"> React is a declarative,component-based Javascript library for building user interfaces.</p> 
-    </div>
-    <div class="techno-content">
+<body>
+  <nav class= "navbar">
+    <img class="image-navbar" src="images/wolox_logo.svg" alt="Wolox logo">
```
Comment (mmamaniwolox) [inline index.html]: Close this and the other img tabs `/>`

File: scss/tech.scss [hunk 8]
```diff
@@ -1,7 +1,7 @@
  @import 'color';
 
 .body {
-  margin: 30px;
+  margin: 0 30px 30px 30px;
```
Comment (mmamaniwolox) [inline scss/tech.scss]: `0 30px 30px` instead

File: scss/titles.scss [hunk 9]
```diff
@@ -4,7 +4,8 @@
   color: $color-black;
   font-size: 24px; 
   position: relative;
-  margin: 80px;
+  margin: 160px 80px 80px 80px;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: `160px 80px 80px` instead

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -8,62 +8,67 @@
...
-      <p class="text-complete"> React is a declarative,component-based Javascript library for building user interfaces.</p> 
-    </div>
-    <div class="techno-content">
-      <img class="img-techno" src="images/angular_icon.svg" alt="React icon">
+<body>
+  <nav class= "navbar">
+    <img class="image-navbar" src="images/wolox_logo.svg" alt="Wolox logo"/>
+    <h1 class="title-navbar">Developer Tools</h1>
+  </nav>
+  <div class="body">
```
Comment (mmamaniwolox) [inline index.html]: Use a more descriptive css class, body is an html element

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

File: scss/navbar.scss [hunk 11]
```diff
@@ -0,0 +1,24 @@
+@import 'color';
+
+.navbar {
+  padding: 10px;
+  top: 0px;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: top: 0;

File: scss/navbar.scss [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+  left: 0px;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: left: 0;

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**